### PR TITLE
feat: product page

### PR DESCRIPTION
**Pull request by shivangisharma01**
Product Page 
image

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
The code looks clean. Good to merge

**Review by Shivanshsharma01 — APPROVED** [review]

**Review by thesudhanshupanda — APPROVED** [review]
good to merge !

File: Page/Cart/cart.css [hunk 1]
```diff
@@ -51,7 +68,59 @@
...
+    padding: 3px 8px;
+    margin: 0 1rem;
+}
+
+.quantity-box {
+    padding: 1rem;
+}
+
+.quantity {
+
```
Comment (vaibhav-mahalle) [inline Page/Cart/cart.css:93]: remove the spaces

File: Page/Login/login.html [hunk 2]
```diff
@@ -44,13 +43,38 @@
...
+                   Don't have a account? <span class="log-btn-link"><a href="/page/Signup/signup.html">Sign Up Here</a></span>
                 </div>
 
-              
             </div>
         </main>
+        
+        </div>
+
+        <div class="section-footer">
```
Comment (ttahm3d) [inline Page/Login/login.html:54]: Is this div adding any value to your code?

**Review by vaibhav-mahalle — APPROVED** [review]
everything else looks good to me.

**Review by ttahm3d — APPROVED** [review]
Looks good. Left a comment. can be merged.

File: Page/Login/login.html (hunk 2 above)
Comment (shivangisharma01) [reply Page/Login/login.html:54]: yes, positioning

**Pull request merged**